README.md

```diff
@@ -16,8 +16,10 @@
 
 ```js
 var Hapi = require('hapi')
-var hoodie = require('@hoodie/server').register
-var PouchDB = require('pouchdb-core').plugin(require('pouchdb-mapreduce')).plugin(require('pouchdb-adapter-memory'))
+var hoodie = require('@hoodie/server')
+var PouchDB = require('pouchdb-core')
+                .plugin(require('pouchdb-mapreduce'))
+                .plugin(require('pouchdb-adapter-memory'))//pouchdb-adapter-memory is optional
```

Comment:
Can you remove the white space? If you want to do line breaks, use 2 spaces of indentation. 

`pouchdb-adapter-memory` is not optional, try this:

```js
const PouchDB = require('pouchdb-core').plugin(require('pouchdb-mapreduce'))
db = new PouchDB('test')
// throws `Error: Invalid Adapter: undefined`
```